README.md

```diff
@@ -106,6 +106,160 @@ of each other.
 - ``example_new_term.yaml`` is an example of the "new term" rule type, which alerts when a new value appears in a field or fields. In this example,
 an email is sent when a new value of ("username", "computer") is encountered in example login logs.
 
+## Frequently Asked Questions
+
+### My rule is not getting any hits?
+
+So you've managed to set up ElastAlert, write a rule, and run it, but nothing happens, or it says ``0 query hits``. First of all, I reccommend using the command ``elastalert-test-rule rule.yaml`` to debug. It will show you how many documents match your filters for the last 24 hours (or more, see ``--help``), and then shows you if any alerts would have fired. If you have a filter in your rule, remove it and try again. This will show you if the index is correct and that you have at least some documents. If you have a filter in Kibana and want to recreate it in ElastAlert, you probably want to use a query string. Your filter will look like
+
+```
+filter:
+- query:
+    query_string:
+      query: "foo: bar AND baz: abc*"
+```
+If you receive an error that Elasticsearch is unable to parse it, it's likely the YAML is not spaced correctly, and the filter is not in the right format. If you are using other types of filters, like ``term``, and common pitfall is not realizing that you may need to use the analyzed token. This is the default if you are using Logstash. For example,
```

Comment:
1. "and common" --> "a common"
2. I think you actually meant to say the not_analyzed token instead.